### PR TITLE
Work in progress: Transaction fees, client-side including analytics

### PR DESCRIPTION
**Pull request by jlieb10**
## Why are you doing this?
We are introducing the ability for contributors to cover transaction fees, pending analysis from finance. This adds the fee to the contribution and sends the fee amount as part of the acquisition data object.

[**Trello Card**](https://trello.com)

## Changes

* Change 1
* Change 2

### CODE REVIEW

File: support-frontend/assets/helpers/tracking/acquisitions.js [hunk 1]
```diff
@@ -223,6 +225,8 @@ const getAbTests = (
 function derivePaymentApiAcquisitionData(
   referrerAcquisitionData: ReferrerAcquisitionData,
   nativeAbParticipations: Participations,
+  baseContributionAmount?: number,
```
Comment (tomrf1) [inline support-frontend/assets/helpers/tracking/acquisitions.js]: does `baseContributionAmount` need to be optional? Shouldn't we always have it at this point?

File: support-frontend/assets/helpers/tracking/acquisitions.js [hunk 2]
```diff
@@ -59,6 +60,7 @@ export type PaymentAPIAcquisitionData = {|
   source: ?string,
   abTests: ?AcquisitionABTest[],
   gaId: ?string,
+  transactionFee?: number,
```
Comment (tomrf1) [inline support-frontend/assets/helpers/tracking/acquisitions.js]: 👍

Comment (tomrf1) [reply support-frontend/assets/helpers/tracking/acquisitions.js]: we'll need to update the model in payment-api before putting this live, but this is the right approach.

File: support-frontend/assets/helpers/tracking/acquisitions.js (hunk 1 above)
Comment (jlieb10) [reply support-frontend/assets/helpers/tracking/acquisitions.js]: nope!

**Pull request closed without merging**